lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -118,6 +118,7 @@
   tags:
     description:
       - A list of tags to add to the Auto Scale Group. Optional key is 'propagate_at_launch', which defaults to true.
+      - If a tag has a colon in it's name, then use double quotes around the tag name.
```

Comment:
This seems like it would apply to all AWS modules that use a dict of tags. I wonder if there's a more visible place for this.

Reply:
There are other things that would need to be quoted too, like the boolean value `false` if you want it to stay a lowercase string and not become `False`. It seems like these tips should be in general docs rather than in one module's documentation.